pkg/providers/v2/cloud.go

```diff
@@ -46,8 +54,22 @@ var _ cloudprovider.Interface = (*cloud)(nil)
 
 // cloud is the AWS v2 implementation of the cloud provider interface
 type cloud struct {
-	creds  *credentials.Credentials
-	region string
+	creds     *credentials.Credentials
+	instances cloudprovider.InstancesV2
+	region    string
+	cfg       *CloudConfig
+	ec2       EC2
+	metadata  EC2Metadata
+}
+
+// CloudConfig wraps the settings for the AWS cloud provider.
+type CloudConfig struct {
```

Comment:
We should remove this if we're not reading it anywhere. I think the use of cluster tags needs a bit more discussion

Reply:
gotcha!